classes/converter.php

```diff
@@ -399,7 +399,10 @@ public function poll_conversion_status(conversion $conversion) {
             $this->delete_converted_file($file->get_pathnamehash());
         } catch (S3Exception $e) {
             $errorcode = $e->getAwsErrorCode();
-            if ($errorcode == 'NoSuchKey') {
+            $timecreated = $conversion->get('timecreated');
+            $timemodified = $conversion->get('timemodified');
+            $timeinprogress = $timemodified - $timecreated;
```

Comment:
Probably we can simplify a bit here, like $timeinprogress = $conversion->get('timemodified') - $conversion->get('timecreated'); But it's my personal preferences. I don't like an additional variables.